passport-auth.js

```diff
@@ -0,0 +1,20 @@
+var passport = require('passport');
+const GoogleStrategy = require('passport-google-oauth20').Strategy;
+
+passport.use(new GoogleStrategy({
+    clientID: process.env.PASSPORT_CLIENT_ID,
+    clientSecret: process.env.PASSPORT_CLIENT_SECRET,
+    callbackURL: "http://localhost:3000/auth/google/callback"
```

Comment:
Seems like we are always redirecting to localhost:3000, which does not work on GCP App Engine. Could you programatically change this based on Node.js environment? Similar to how we decide database collection name by environments:  

```const DB_COLLECTION_NAME = (process.env.NODE_ENV == "production") ? 'movies' : 'dev-movies';```